integration-test-skip-prepared-statement-values/src/test/java/com/ericsson/bss/cassandra/ecaudit/integration/skippreparedstatementvalues/ITDataAuditSkipPreparedStatementValues.java

```diff
@@ -0,0 +1,140 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecaudit.integration.skippreparedstatementvalues;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.LoggerFactory;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.Appender;
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.SimpleStatement;
+import com.ericsson.bss.cassandra.ecaudit.logger.Slf4jAuditLogger;
+import com.ericsson.bss.cassandra.ecaudit.test.daemon.CassandraDaemonForAuditTest;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ITDataAuditSkipPreparedStatementValues
+{
+    private static Cluster cluster;
+    private static Session session;
+
+    @Captor
+    private ArgumentCaptor<ILoggingEvent> loggingEventCaptor;
+
+    @Mock
+    private Appender<ILoggingEvent> mockAuditAppender;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception
+    {
+        CassandraDaemonForAuditTest cdt = CassandraDaemonForAuditTest.getInstance();
+        cluster = cdt.createCluster();
+        session = cluster.connect();
+    }
+
+    @Before
+    public void before()
+    {
+        LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
+        loggerContext.getLogger(Slf4jAuditLogger.AUDIT_LOGGER_NAME).addAppender(mockAuditAppender);
+    }
+
+    @After
+    public void after()
+    {
+        verifyNoMoreInteractions(mockAuditAppender);
+        LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
+        loggerContext.getLogger(Slf4jAuditLogger.AUDIT_LOGGER_NAME).detachAppender(mockAuditAppender);
+    }
+
+    @AfterClass
+    public static void afterClass()
+    {
+        session.close();
+        cluster.close();
+    }
+
+    @Test
+    public void testSuccessfulStatements()
```

Comment:
One way to save some build/test time and minimize repetitive code would be to move this particular test case into the IT for custom format (and perhaps call it testNakedSuccessfulStatements(), and then simply modify the audit.yaml in it-custom-format to use OPERATION_NAKED.

Reply:
removed the extra it module and moved the test into the custom-format it